com.unity.netcode.gameobjects/Runtime/Core/NetworkManager.cs

```diff
@@ -1054,18 +1138,15 @@ private void OnNetworkManagerTick()
 
         private void SendConnectionRequest()
         {
-            var clientIds = new[] { ServerClientId };
-            var context = MessageQueueContainer.EnterInternalCommandContext(MessageQueueContainer.MessageType.ConnectionRequest, NetworkDelivery.ReliableSequenced, clientIds, NetworkUpdateStage.EarlyUpdate);
-            if (context != null)
-            {
-                using var nonNullContext = (InternalCommandContext)context;
-                nonNullContext.NetworkWriter.WriteUInt64Packed(NetworkConfig.GetConfig());
-
-                if (NetworkConfig.ConnectionApproval)
-                {
-                    nonNullContext.NetworkWriter.WriteByteArray(NetworkConfig.ConnectionData);
-                }
-            }
+            var message = new ConnectionRequestMessage
+            {
+                ConfigHash = NetworkConfig.GetConfig(),
+                ShouldSendConnectionData = NetworkConfig.ConnectionApproval,
+                ConnectionData =
```

Comment:
we don't have any specific max line length limit in the standards, why are you breaking lines into multiple? 👀

Reply:
I think Rider did this for me and I didn't change it. (But, I use portrait monitors, so a little allowance for breaking things up to avoid horizontal scrolling would be appreciated. 🙏 But I also recognize that's a "me" thing.)

Reply:
Rider is really awful about how it breaks up lines. My personal recommendation is that if you are going to break up part of the lines, break up all expressions. So my suggestion is that it should be
```
ConnectionData =
    new FixedUnmanagedArray<byte, ConnectionRequestMessage.ConnectionDataStorage>(
        NetworkConfig.ConnectionData);
```
Especially because dissociating `ConnectionData` from `NetworkConfig` makes readability much _worse_ not better.

Reply:
Done.